CONTRIBUTING.md

```diff
@@ -301,37 +243,83 @@ This is optional but encouraged.
 Good commit messages explain what changed AND why you changed it.
 See [I've written a clear changelist description](https://github.com/camunda/camunda/wiki/Pull-Requests-and-Code-Reviews#ive-written-a-clear-changelist-description).
 
-## Contributor License Agreement
 
-You will be asked to sign our Contributor License Agreement when you open a Pull Request. We are not
-asking you to assign copyright to us but to give us the right to distribute
-your code without restriction. We ask this of all contributors to
-assure our users of the origin and continuing existence of the code. You only
-need to sign the CLA once.
+## Build Camunda from source
 
-## Licenses
+Camunda is a multi-module Maven project. To **quickly** build all components,
+run the command: `mvn clean install -Dquickly` in the root folder.
 
-Zeebe, Operate, and Tasklist source files are made available under the
-[Camunda License Version 1.0](/licenses/CAMUNDA-LICENSE-1.0.txt) except for the parts listed
-below, which are made available under the [Apache License, Version
-2.0](/licenses/APACHE-2.0.txt).  See individual source files for details.
+> [!NOTE]
+> All Java modules in Zeebe are built and tested with JDK 21. Most modules use language level
+> 21, exceptions are: zeebe-bpmn-model, zeebe-client-java, zeebe-gateway-protocol,
+> zeebe-gateway-protocol-impl, zeebe-protocol and zeebe-protocol-jackson which use language level 8
```

Comment:
❓ Should we also document the JDK requirements for Operate/Tasklist/etc?

Reply:
@korthout I'd assume that we now be design have the same JDK for the core modules being Zeebe, Operate, Tasklist (and Optimize in the near future). Hence I propose to just state 

```suggestion
> All Camunda core modules are built and tested with JDK 21. Most modules use language level
> 21, exceptions are: zeebe-bpmn-model, zeebe-client-java, zeebe-gateway-protocol,
> zeebe-gateway-protocol-impl, zeebe-protocol and zeebe-protocol-jackson which use language level 8
```